src/configs/chalk.c42spec

```diff
@@ -1992,27 +1992,27 @@ end users.
     type:    string
     default: "/sys/class/dmi/id/board_vendor"
     hidden:  true
-    doc:     "Path where to check AWS board vendor if running in AWS EC2"
+    doc:     "Path where to check AWS board vendor if running in AWS, Google or Microsoft nodes"
   }
 
   field sys_product_path {
     # https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/identify_ec2_instances.html
     type:    string
     default: "/sys/devices/virtual/dmi/id/product_uuid"
     hidden:  true
-    doc:     "Path where to check AWS product if running in AWS EC2"
+    doc:     "Path where to check product uuid for cloud nodes"
   }
 }
 
-singleton aws {
-  gen_fieldname: "awsConfig"
-  gen_typename:  "AwsConfig"
+singleton cloud_provider {
+  gen_fieldname: "cloudProviderConfig"
+  gen_typename:  "CloudProviderConfig"
   gen_setters:   false
   user_def_ok:   false
   shortdoc: """
-Configuration information for the AWS Cloud Provider
+Configuration information for the different Cloud Provider
 """
-  allow ec2
```

Comment:
similar here. I think `ec2` is clear in aws context but `cloud_instance_hw_identifiers` I dont know what that means without looking at all the keys

Reply:
the /sys/vendor bits is generic Linux functionality so not tied to ec2